braket_features/Getting devices and checking device properties.ipynb

```diff
@@ -0,0 +1,340 @@
+     ]
+    }
+   ],
+   "source": [
+    "# the IonQ device\n",
+    "device = AwsDevice(\"arn:aws:braket:::device/qpu/ionq/ionQdevice\")\n",
+    "\n",
+    "# the Rigetti device\n",
+    "device = AwsDevice(\"arn:aws:braket:::device/qpu/rigetti/Aspen-8\")\n",
+    "\n",
+    "execution_windows = device.properties.service.executionWindows[0]\n",
+    "connectivity_graph = device.properties.paradigm.connectivity\n",
+    "provider_information = device.properties.provider\n",
+    "\n",
+    "print(f'The exeuation windows of the {device.name}:\\n{execution_windows}')\n",
+    "print('\\n The connectivity graph of the qubits for this device:\\n', connectivity_graph)\n",
```

Comment:
Remove space after first \n. Ditto below.

Reply:
there is no space in the original ipynb file

Reply:
I do see a space in the notebook you sent me.